docs/section6/Rule6-7.md

```diff
@@ -0,0 +1,60 @@
+
+# Lighting - Rule 6-7
+
+**Rule ID:** 6-7  
+**Rule Description:** Where a complete lighting system exists and where a lighting system has been designed and submitted with design documents, the baseline LPD is equal to expected value in Table G3.7. Where lighting neither exists nor is submitted with design documents, baseline LPD shall be determined in accordance with Table G3-7 for "Office-Open Plan" space type.
+
```

Comment:
The second sentense should say "Where lighting neither exists nor is submitted with design documents and **lighting space type is unknown**, baseline LPD shall be determined in accordance with Table G3-7 for "Office-Open Plan" space type.